artifactory/users.go

```diff
@@ -65,10 +65,8 @@ func (d DeleteAPIKey) String() string {
 }
 
 // GetAll returns a list of all users.
-//
-// Docs: https://www.jfrog.com/confluence/display/RTF/Artifactory+REST+API#ArtifactoryRESTAPI-GetUsers
 func (s *UsersService) GetAll() (*[]User, *Response, error) {
-	u := fmt.Sprintf("/api/security/users")
+	u := fmt.Sprintf("/api/users")
```

Comment:
This updated endpoint better reflects the [User struct](https://github.com/target/go-arty/blob/master/artifactory/users.go#L32).